openbas-api/src/main/java/io/openbas/rest/mapper/form/MapperUpdateInput.java

```diff
@@ -0,0 +1,26 @@
+package io.openbas.rest.mapper.form;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+import static io.openbas.config.AppConfig.MANDATORY_MESSAGE;
+
+@Getter
+@Setter
+public class MapperUpdateInput {
```

Comment:
This is the same POJO as MapperAddInput right ?
Can we have one MapperInput POJO ?

Same question for others

Reply:
It's true for mapper that both have the same properties but not the others as the Inject Importer and Rule Attribute update POJOs have the id in case we're modifying an existing one and not creating a new one. 
At first, I thought about having the same object too but I thought it would be more in line with what we can find for other rest service (see document for instance).